DRAFT_trophic_levels-biomass_indicators.rmd

```diff
@@ -0,0 +1,424 @@
+
+#Expected carnivore biomass 
+#Taken from Hatton et al. Figure 1
+expected_carnivore_biomass<-(totalherbivorebiomass^0.73)*0.014#African carnivore and herbivore communities
+ggplot()+geom_spatraster(data=expected_carnivore_biomass)+
+  ggtitle("Expected carnivore biomass")+
+  facet_wrap(~lyr)+theme_bw()+
+  scale_fill_continuous(low='lightskyblue',high="darkblue",na.value=NA,expression('kg km'^-2),trans='log10')
+
+diffcarn<-100* (totalcarnivores-expected_carnivore_biomass)/(expected_carnivore_biomass+1)
+names(diffcarn)<-allyears_vect
+
+ggplot()+geom_sf(data=norcounty,fill="white",lwd=0.1)+
+  geom_spatraster(data=diffcarn)+
+  ggtitle('Difference from expected carnivore biomass')+
+  scale_fill_distiller(type="div",limit=c(-100,100),oob=squish,na.value=NA,"%difference from \nexpected biomass")+
+  facet_wrap(~lyr)+theme_bw()
+
+#writeRaster(expected_carnivore_biomass,paste0(pData,"/Trophic_levels/expectedcarnivorebiomass.tif"),overwrite=TRUE)
+
+#writeRaster(diffcarn,paste0(pData,"/Trophic_levels/CarnivoreBiomassDiffExpected.tif"),overwrite=TRUE)
+
+```
+
+From the above we see that Norway has had less carnivore biomass than expected based on the available herbivore biomass across the time period. Some counties and time periods (e.g. Hedmark today and Finnmark in 1907 had about the same carnivore biomass as predicted.
+
+## Part 2 - Habitat specific biomass distributions
+
+We now create biomass ratios for specific habitat types. To do this we need an ecosystem map and to place each consumer species within each habitat.
+
+Indicators were not developed for mire ecosystems. This decision was taken in conversation with the mire ecosystem group, on the basis that herbivores have minimum use of the mrie ecosystems.
+
+Table of the consumer species associated with each habitat.
+
+| Ecosystem             | Livestock               | Wild herbivores   | Carnivores        |
+|------------------|-------------------|------------------|------------------|
+| Forest                | Storfe                  | Elg, rådyr, hjort | Ulv, gaupe, bjørn |
+| Alpine                | Sau, tamrein, geit      | Villrein          | Jerv              |
+| Semi-naturlig mark    | Sau, hest, geit, storfe | Hjort             | Gaupe, ulv        |
+| Naturlig åpne områder | Sau, geit, storfe       | Hjort             | Gaupe, ulv        |
+| Myr                   | \-\--                   | \-\--             | \-\--             |
+
+Since a suitable ecosystem map is still under development, we use the AR50 land-cover map as a place-holding solution. AR50 does not give a good representation of naturlig åpen områder or semi-naturlig mark, so we demonstrate the habitat specific biomass distributions using the forest class in AR50 and the species outlined above.
+
+```{r}
+#| label: Habitat specific rasters
+#| cache: TRUE
+
+#Forest
+#AR50 type raster
+artype50<-rast("/data/R/GeoSpatialData/LandCover/Norway_LandResource_AR50/Processed/Raster/AR50_Norge.tif")
```

Comment:
It all works to here and then no longer works - I guess I have replaced the wrong data - can we get the original from James? (look on his qmd file as I have changed it here)